### ISSUE TITLE
pointless additional default route

### ISSUE THREAD

**Issue by pisto** [online-labs/image-fedora#18]
It seems that you somewhere inject an additional default route:
```
default via 10.1.4.1 dev eth0  proto static  metric 1024
default via 10.1.4.1 dev eth0
```
the second one seems unnecessary.

On a related note, I spent some time with edouardb fixing a tcp slowness on my instance, and in the end it turned out that it can be fixed by setting the mtu for the default path to 1500. I know that you're switching to 1500 mtu everywhere, but you may consider setting the mtu to 1500 with dhcp only for the default route.

**Comment by moul** [comment]
Hum, we are not setting the mtu to 1024, as you said we are sending the mtu using DHCP, and it should be 1500

Do you think this script https://github.com/online-labs/image-fedora/blob/453960632985aa2b8b455c9eaa33b250205b3a5e/patches/etc/sysconfig/network-scripts/ifcfg-eth0 has something bad ?

Do you know if fedora could set the mtu to 1024 for any reason ?

**Comment by moul** [comment]
Thanks again for all your feedbacks, we are not fedora gurus and greatly appreciate your help here

**Comment by pisto** [comment]
No, "metric 1024" is something set by the kernel when an automatically configured route goes up (dhcp), it mostly has no significance as far as I know. The second route, lacking that, seems to be manually added by some config. The eth0 ifcfg doesn't seem to be the cause. I used the 1500 mtu to replace the 9000 mtu, and that fixed my problem.

**Comment by pisto** [comment]
To clarify this is what my routing table looks like now
```
default via 10.1.4.1 dev eth0  proto static  metric 1024  mtu 1500
10.1.4.0/23 dev eth0  proto kernel  scope link  src 10.1.5.68 
```

**Comment by moul** [comment]
```
default via 10.1.2.1 dev eth0
default via 10.1.2.1 dev eth0  proto static  metric 1024
10.1.2.0/23 dev eth0  proto kernel  scope link  src 10.1.3.57
10.1.31.34 via 10.1.2.1 dev eth0  proto static  metric 1
```

The two static routes are set during the boot process (in the bios equivalent), on other images (ubuntu for instance), the dhcp script will erase the static routes, but apparently not in fedora

Even if it is unnecessary, do you see any problem it can come with ? if so, what do you think is the best solution ?

**Comment by pisto** [comment]
It doesn't seem to be harmful. I looked at the ifcfg scripts and it seems they should replace the default route, so dunno.
